content/installation/image-builder.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: bt_wiki
+title: Using the Image Builder
+category: Installation
+draft: false
+weight: 600
+
+---
+
+{{% gsNote title="Supported Clouds" %}}
+ * AWS
+ * OpenStack
+ * vSphere
+{{% /gsNote %}}
+
+{{% gsNote title="Prerequisites" %}}
+ * [packer](https://www.packer.io/intro/getting-started/setup.html) += 0.8.6 (older versions not tested)
+ * Account credentials for each cloud platform you want to build on.
+{{% /gsNote %}}
+
+
+ 1. Clone the image bakery repo
+ 2. cd into the image bakery path
+    ```bash
+    git clone https://github.com/cloudify-cosmo/cloudify-image-bakery.git
+    cd cloudify-image-bakery/cloudify_marketplace
+    ```
+
+ 3. Copy inputs-example.json and edit with your favourite editor
+    ```bash
+    cp inputs-example.json inputs.json
+    $EDITOR inputs.json
+    ```
+
+    Point `packer` at a CentOS 7(TODO?) image in your cloud:
+    ```json
+    "aws_source_ami": "ami-91feb7fb",
+    "openstack_image_flavor": "d87de0ca-9c0e-4759-a704-8621883c3415",
+    ```
+
+  {{% gsNote title="Multiple Clouds" %}}
+  Fill in the fields for each cloud platform you intend to use in the same file.
+  If you don't plan to use OpenStack for example you can leave the 'openstack_' prefixed fields as they are.
+  {{% /gsNote %}}
+
+ 4. Run `packer`
+    ```bash
+    $ packer build --only=aws,openstack --var-file=inputs.json cloudify.json
```

Comment:
Explain the command and how the user is likely to want to change it?
(e.g. --only= can be changed to install only on one specific cloud such as --only=aws)